.github/DISCUSSION_TEMPLATE/czi-compress-questions.yml

```diff
@@ -0,0 +1,10 @@
+---
+labels: ["czicompress", "question"]
+body:
+  - type: textarea
+    id: question
+    attributes:
+      label: Question
+      description: Please add any screenshots of logs as applicable
```

Comment:
OR  logs?

Reply:
Added in [57dc64d](https://github.com/ZEISS/czicompress/pull/45/commits/57dc64dd9f006ba1327b42eadba31d0f50d450ab)